lib/header.js

```diff
@@ -49,20 +49,9 @@ function linksHandler(req, res, next) {
     }
 
     // Add ACL and Meta Link in header
-    var aclLink = utils.getResourceLink(
-        filename, uri,
-        ldp.root, ldp.suffixAcl,
-        ldp.suffixMeta);
-
-    var metaLink = utils.getResourceLink(
-        filename, uri,
-        ldp.root, ldp.suffixMeta,
-        ldp.suffixAcl);
-
-    addLink(res, utils.uriRelative(aclLink), 'acl');
-    addLink(res, utils.uriRelative(metaLink), 'describedBy');
-
-
+    addLink(res, utils.basename(req.url) + ldp.suffixAcl, 'acl');
```

Comment:
I'm inclined to deprecate it, but let's give it a bit of time to see if we're not missing anything.